src/app/Rules/CreateRule.tsx

```diff
@@ -185,6 +208,7 @@ const Comp = () => {
 
   return (
     <BreadcrumbPage pageTitle='Create' breadcrumbs={breadcrumbs} >
+      <Button onClick={() => {context.api.doGet<EventTemplate[]>(`targets/localhost:0/templates`).subscribe()}}/>
```

Comment:
This looks like an addition for manual testing or something, not something actually needed for the feature, right?

Reply:
Ahh yea, I really forgot about that.